documentation/book/con-partition-reassignment.adoc

```diff
@@ -81,3 +81,32 @@ The following is an example reassignment JSON file that assigns topic `topic-a`,
 
 Partitions not included in the JSON are not changed.
 
+== Reassigning partitions between JBOD volumes
```

Comment:
My understanding of this chapter is that you are referring to the `kafka-reassign-partitions.sh` tool. Is that right?
If yes, should be mentioned here? If not, I haven't understood where the described JSON should be used.

Reply:
It is part of a chapter which is called _Partition reassignment_ and is whole about the `kafka-reassign-partitions.sh`. When you look at the whole chapter / module and not just at the small subsection, do you think it is needed to make it more explicit what is it talking about?

Reply:
You are right, I didn't "explode" the previous part and noticed that it was part of a greater chapter :-)